crates/api_models/src/payments.rs

```diff
@@ -4747,6 +4774,20 @@ pub struct GpaySessionTokenData {
     pub data: GpayMetaData,
 }
 
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+pub struct SamsungSessionTokenData {
+    #[serde(rename = "samsung_pay")]
+    pub data: SamsungMetaData,
+}
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+pub struct SamsungMetaData {
+    pub service_id: String,
```

Comment:
We can wrap this within `Secret` if this is considered sensitive information.

Reply:
> We can wrap this within `Secret` if this is considered sensitive information.

No, it is not a secret value. We send this is session token response to sdk in order to make call to Samsung Pay.